source/optional_features/custom_error_pages/index.html.md.erb

```diff
@@ -0,0 +1,24 @@
+---
+title: Custom error pages 
+weight: 135
+---
+
+# Custom error pages
+
+GOV.UK Pay supports custom error pages.
```

Comment:
I'd prefer this to be more explicit about the behaviour that the user will see, something along the lines of, "you can use your own pages for unsuccessful payments"

Reply:
That's fine—as I suggest in the other comment there's no real reason why we shouldn't use that as the title for the section.